src/index.html

```diff
@@ -8,7 +8,50 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <header class="container">
+    <div class="flex_container">
+      <a href="#home" class="logo_link">
+        <img
+        src="/images/logo.png"
+        alt="Logo"
+        class="logo flex_item">
+      </a>
+    </div>
+    <nav class="nav">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
    </div>
    
    <nav class="nav">
```